app/migrations/Version20170419153243.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20170419153243 extends AbstractMigration
```

Comment:
Done, but note that this should not cause any problem since versions are sorted in a db table.